packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -3,7 +3,7 @@
     <div v-show="localMinimized" class="morph">
       <button :class="['morph-display-wrapper', 'btn', btnType, 'card-title']" @click="open()">
         <slot name="_alt">
-          <slot name="header"></slot>
```

Comment:
hmm what's this for? seeing that we wouldn't have a panel expanded but minimized at the same time

Reply:
Yes this is not needed, thanks for the catch!